homeassistant/components/config/config_entries.py

```diff
@@ -587,3 +639,61 @@ async def _async_matching_config_entries_json_fragments(
         )
         or (filter_is_not_helper and entry.domain not in integrations)
     ]
+
+
+@websocket_api.require_admin
+@websocket_api.websocket_command(
+    {
+        "type": "config_entries/subentries/list",
+        "entry_id": str,
+    }
+)
+@websocket_api.async_response
+async def config_subentry_list(
+    hass: HomeAssistant,
+    connection: websocket_api.ActiveConnection,
+    msg: dict[str, Any],
+) -> None:
+    """List subentries of a config entry."""
+    entry = get_entry(hass, connection, msg["entry_id"], msg["id"])
+    if entry is None:
+        return
```

Comment:
we should return an error with `NOT_FOUND`

Reply:
This confused me too, but the function `get_entry` actually sends an error message: https://github.com/home-assistant/core/blob/a6c541a4a5ccdd3eec261d0612ec5d7f454c3a83/homeassistant/components/config/config_entries.py#L371-L380

We could, in a separate PR, rename the `get_entry` function so it's clearer that's how it works.